.github/workflows/deploy-to-pages.yml

```diff
@@ -0,0 +1,49 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Deploy to pages
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [master]
+  pull_request:
+    branches: [feature/deploy-to-pages-action]
```

Comment:
So this needs to be changed to master now that it's a real PR

Reply:
Ok great I think I got the flow down:

- change pull_request to whatever PR it is and open a PR on it to get it to run
- if fails fix, if not change pull_request to master and make another PR

Going to change and open a new one now